Scripts/Python/QueryGroupManagementOmeREDFISH.py

```diff
@@ -0,0 +1,706 @@
+Status: 3000
+ConnectionState: True
+AssetTag: None
+SystemId: 1879
+DeviceName: WIN-4UI980INRK9
+LastInventoryTime: 2020-03-16 05:00:55.439
+LastStatusTime: 2020-03-16 20:00:04.329
+DeviceSubscription: None
+DeviceCapabilities: [1, 2, 3, 4, 5, 6, 7, 8, 9, 41, 11, 12, 13, 14, 15, 16, 1009, 17, 50, 18, 30, 31]
+SlotConfiguration: {'ChassisName': None}
+DeviceManagement: [{'ManagementId': 5004, 'NetworkAddress': '100.65.84.100', 'MacAddress': '10:98:36:b2:06:cd', 'ManagementType': 2, 'InstrumentationName': 'idrac-M537C3S', 'DnsName': 'idrac-M537C3S', 'ManagementProfile': [{'ManagementProfileId': 5004, 'ManagementId': 5004, 'AgentName': 'iDRAC', 'Version': '4.00.00.00', 'ManagementURL': 'https://100.65.84.100:443', 'HasCreds': 0, 'Status': 1000, 'StatusDateTime': '2020-03-10 03:43:20.994'}]}]
+Actions: None
+SensorHealth@odata.navigationLink: /api/DeviceService/Devices(10073)/SensorHealth
+VirtualSession: {'@odata.id': '/api/DeviceService/Devices(10073)/VirtualSession'}
+Baselines: {'@odata.id': '/api/DeviceService/Devices(10073)/Baselines'}
+InventoryDetails@odata.navigationLink: /api/DeviceService/Devices(10073)/InventoryDetails
+HardwareLogs@odata.navigationLink: /api/DeviceService/Devices(10073)/HardwareLogs
+SubSystemHealth@odata.navigationLink: /api/DeviceService/Devices(10073)/SubSystemHealth
+RecentActivity@odata.navigationLink: /api/DeviceService/Devices(10073)/RecentActivity
+InventoryTypes: {'@odata.id': '/api/DeviceService/Devices(10073)/InventoryTypes'}
+LogSeverities: {'@odata.id': '/api/DeviceService/Devices(10073)/LogSeverities'}
+Settings@odata.navigationLink: /api/DeviceService/Devices(10073)/Settings
+Temperature: {'@odata.id': '/api/DeviceService/Devices(10073)/Temperature'}
+Power: {'@odata.id': '/api/DeviceService/Devices(10073)/Power'}
+SystemUpTime: {'@odata.id': '/api/DeviceService/Devices(10073)/SystemUpTime'}
+BlinkStatus: {'@odata.id': '/api/DeviceService/Devices(10073)/BlinkStatus'}
+PowerUsageByDevice@odata.navigationLink: /api/DeviceService/Devices(10073)/PowerUsageByDevice
+DeviceBladeSlots@odata.navigationLink: /api/DeviceService/Devices(10073)/DeviceBladeSlots
+GraphicInfo: {'@odata.id': '/api/DeviceService/Devices(10073)/GraphicInfo'}
+DeployRequired: {'@odata.id': '/api/DeviceService/Devices(10073)/DeployRequired'}
+
+
+9.	Script also supported deleting a query group. Use -D argument to delete a query group.
+
+C:\Python38-32> QueryGroupManagementOmeREDFISH.py -ip 192.168.0.120 -u admin -p admin -D 10495
+
+- WARNING, executing POST command to delete group ID 10495
+- PASS, POST command passed to delete group ID 10495
+- WARNING, executing GET to validate group ID 10495 no longer exists
+- PASS, validation of group ID 10495 no longer exist for OME"""
+
+    print(help_text)
+    with open("help_text_workflow.txt","w") as x:
+        x.writelines(help_text)
+    print("-"*100)
+    print("\n- WARNING, end 2 end workflow example also captured in \"help_text_workflow.txt\" file")
+
+
+
+
+
+def get_devices():
```

Comment:
I don't believe this handles pagination - the https://github.com/dell/OpenManage-Enterprise/blob/master/Core/Python/get_device_list.py script should have a working example of this